database/staking_test.go

```diff
@@ -174,11 +224,21 @@ func (suite *DbTestSuite) TestBigDipperDb_GetValidatorsData() {
 			"cosmosvalcons1qq92t2l4jz5pt67tmts8ptl4p0jhr6utx5xa8y",
 			"cosmosvaloper1000ya26q2cmh399q4c5aaacd9lmmdqp90kw2jn",
 			"cosmosvalconspub1zcjduepqe93asg05nlnj30ej2pe3r8rkeryyuflhtfw3clqjphxn4j3u27msrr63nk",
+			"",
+			"",
+			"",
+			"",
+			"",
```

Comment:
I think we should specify some values here to be sure they are properly stored and later read

Reply:
Since GetValidatorData means to get back consensus_address, consensus_pubkey,operator_address only, we are not interested in the descriptions. Also there are another method to get back the details, so we dont need to specify this in this test.

Reply:
Actually, `GetValidatorData` returns a `types.Validator` object which is defined as follows: 

```golang
// Validator represents a single validator.
// This is defined as an interface so that we can use the SDK types
// as well as database types properly.
type Validator interface {
	GetConsAddr() sdk.ConsAddress
	GetConsPubKey() crypto.PubKey
	GetOperator() sdk.ValAddress	
	GetDescription() staking.Description
}
```

For this reason, we need to test that also the `GetDescription` method returns the proper data.